lib/register.js

```diff
@@ -17,7 +17,10 @@ const proxyHandler = {
     if (name === Symbol.toStringTag) {
       return 'Module'
     }
-    return getters.get(target)[name]()
+
+    if (typeof getters.get(target)[name] === "function") {
+      return getters.get(target)[name]();
```

Comment:
Couple of linter issues to fix here

```suggestion
    if (typeof getters.get(target)[name] === 'function') {
      return getters.get(target)[name]()
```